fitbenchmarking/results_processing/tests/test_performance_profiler.py

```diff
@@ -313,15 +303,61 @@ def test_get_plot_path_and_data_returns_dict_for_data(self):
                      'dfogn [dfo]': [0.6, 0.4, 0.6, 0.9, 0.5,  0.3, 0.2]}
 
         bounds = {'acc': acc, 'runtime': runtime, 'emissions': emissions}
-        _, pp_dfs = performance_profiler.\
-            get_plot_path_and_data(bounds,
-                                   self.fig_dir,
-                                   self.options)
+        paths = performance_profiler.get_plot_path(bounds,
+                                                   self.fig_dir,
+                                                   self.options)
+        expec_paths = {}
+        for name, _ in bounds.items():
+            this_filename_html = os.path.join(self.fig_dir,
+                                              f"{name}_profile.html")
```

Comment:
I mentioned this but just so everything is on the review, I'd be careful that tests aren't the same code as what they're testing. This should work (it's quite simple) but it may be better to write out the expected paths in full.